### PR TITLE
New version 

### PR DESCRIPTION
**Pull request by Preethi-Dev**
Create a brand new index.html. Hope you love this

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (Preethi-Dev) [inline index.html:7]: Change to

<title>Home page</title>

**Review by Preethi-Dev — COMMENTED** [review]
change the title

File: index.html (hunk 1 above)
Comment (twin-engineers) [reply index.html:7]: yaa good change

**Review by twin-engineers — CHANGES_REQUESTED** [review]
change to 

`<title>home page</title>`

**Pull request merged**